webapp/channels/src/components/channel_header_dropdown/menu_items/view_pinned_posts/view_pinned_posts.test.tsx

```diff
@@ -21,12 +21,12 @@ describe('components/ChannelHeaderDropdown/MenuItem.ViewPinnedPosts', () => {
     };
 
     it('should match snapshot', () => {
-        const wrapper = shallow<ViewPinnedPosts>(<ViewPinnedPosts {...baseProps}/>);
+        const wrapper = shallow<typeof ViewPinnedPosts>(<ViewPinnedPosts {...baseProps}/>);
```

Comment:
The type should not be needed in these tests, so you can remove everything in between the `<>`. (i.e. `shallow(<ViewPinnedPosts {...baseProps}/>`)